swagger_spec_validator/common.py

```diff
@@ -51,6 +56,11 @@ def read_file(file_path):
     return read_url(get_uri_from_file_path(file_path))
 
 
+@lru_cache(maxsize=12)
+def read_file_and_cache(file_path):
+    return read_file(file_path)
```

Comment:
`swagger_spec_validator.validator20` invokes this method with a similar code
```
path = resource_filename(resource_path)
value = read_file_and_cache(path)
```
I'm wondering if it would be easier from a maintenance and understanding perspective if we would have a method like
```python
@lru_cache(maxsize=12)
def read_resource_file(resource_path):
    return read_file(resource_filename(resource_path))
```
In this case `maxsize` is not strictly required anymore as only resource files might be cached

Reply:
I like it.